Integrations/integration-C2sec.yml

```diff
@@ -0,0 +1,236 @@
+description: Understand Your Cyber Exposure as Easy as a Google Search
+configuration:
+- display: URL
+  name: endpointURL
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+script:
+  script: |-
+    import requests
+    import json
+    import datetime
+
+
+    def addcompany(domain, newscan):
+       if newscan == 'true':
+         newscan = 1
+       else:
+           newscan = 0
+       params = {'apikey': demisto.params()['apikey'], 'domain': domain, 'newscan': newscan}
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(demisto.params()['endpointURL'] + 'iriskaddcompany', headers=headers, params=params)
+       if call.status_code == requests.codes.ok:
+          md = tableToMarkdown('Add Company Result', call.json())
+          return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {}
+                }
+       else:
+           return ('Error adding a new company [%d] - %s' % (call.status_code, call.text))
+
+
+    def queryWorkitem(workitemid):
+       params = {'apikey': demisto.params()['apikey'], 'workitemid': workitemid}
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(demisto.params()['endpointURL'] + '/iriskqueryapistatus', headers=headers, params=params)
```

Comment:
in the other commands you didn't use `/` so you must have got mistake in one of the options + please make sure in the CONST creation for the endpointURL that you either have a `/` in the end or not, and then follow your convention in the get calls